boards/px4/fmu-v5x/init/rc.board_defaults

```diff
@@ -16,7 +16,7 @@ param set-default SENS_EN_INA238 0
 param set-default SENS_EN_INA228 0
 param set-default SENS_EN_INA226 1
 
-if ver hwbasecmp 008 009 00a 010
+if ver hwbasecmp 008 009 00a 010 011
```

Comment:
Sorry @niklaut I should have been clearer : I was revering to here.

Reply:
ah, so `hwbasecmp` expects decimal values? Cos then `00a` is wrong too. That's why assumed it was hexadecimal.

Reply:
better check it.

Reply:
it my be hex

Reply:
Are you sure it's not hexadecimal? The `ver hwbasecmp` does a `strcmp`, so it takes a string as input:
https://github.com/PX4/PX4-Autopilot/blob/28db3e1c8ca97fd1bb6067be15c9caf19db8e735/src/systemcmds/ver/ver.cpp#L147

It compares against `px4_board_base_type()` which resolves to the `hw_base_info` char array.
https://github.com/PX4/PX4-Autopilot/blob/28db3e1c8ca97fd1bb6067be15c9caf19db8e735/platforms/nuttx/src/px4/nxp/imxrt/board_hw_info/board_hw_rev_ver.c#L302

which is formatted here
https://github.com/PX4/PX4-Autopilot/blob/28db3e1c8ca97fd1bb6067be15c9caf19db8e735/platforms/nuttx/src/px4/nxp/imxrt/board_hw_info/board_hw_rev_ver.c#L409

with a `%03x` specifier:
https://github.com/PX4/PX4-Autopilot/blob/28db3e1c8ca97fd1bb6067be15c9caf19db8e735/platforms/nuttx/src/px4/common/include/px4_platform/board_determine_hw_info.h#L41

https://github.com/PX4/PX4-Autopilot/blob/28db3e1c8ca97fd1bb6067be15c9caf19db8e735/platforms/common/include/px4_platform_common/board_common.h#L285

So I think this is correct, the code does the right thing and the debugger also agrees with me:
```
(gdb) p hw_base_info
$1 = "011\000\000\000\000\000\000"
```

Reply:
OK - I wrote it and did not remenber!

Reply:
happens to the best of us ;-P